tx_list.go

```diff
@@ -53,15 +53,34 @@ func (tx *Tx) RPeek(bucket string, key []byte) (item []byte, err error) {
 		return nil, ErrBucket
 	}
 
+	if tx.CheckExpire(bucket, key) {
+		return nil, ErrKeyNotFound
+	}
+
 	item, _, err = tx.db.ListIdx[bucket].RPeek(string(key))
 
 	return
 }
 
 // push sets values for list stored in the bucket at given bucket, key, flag and values.
 func (tx *Tx) push(bucket string, key []byte, flag uint16, values ...[]byte) error {
+	var (
+		Expired   bool
+		ttl       uint32
+		timestamp uint64
+	)
+	Expired = false
+	ttl = Persistent
+	_, isExist := tx.db.ListIdx[bucket]
+	if isExist {
+		ttl, Expired = tx.db.ListIdx[bucket].TTL[string(key)]
+	}
+	timestamp = uint64(time.Now().Unix())
+	if Expired {
```

Comment:
This is to express whether the TTL of the list exists, not to express the meaning of expiration ？Whether the variable Expired can express meaning？